rakelib/plugins-metadata.json

```diff
@@ -121,7 +121,8 @@
   },
   "logstash-filter-elastic_integration": {
     "default-plugins": false,
```

Comment:
Once we solve `TarWriter` 256 path limit issue, we can flip this to true and see. The dependant PR to shorten the path: https://github.com/elastic/logstash-filter-elastic_integration/pull/114